content/docs/hosting/BYOC.md

```diff
@@ -98,169 +99,178 @@ With Appbase.io Dashboard you can easily share cluster with your team members. E
 
--   **Step 4 -** Restart Arc Service
+**Example**
 
 ```bash
-systemctl restart arc
+curl -u="USERNAME:PASSWORD" http://EC2-PUBLIC-[IP/DNS]
 ```
 
+#### **Accessing Arc Dashboard**
+
+Now that all our configurations are completed, in order to access all the Arc Services, let us Sign in to [Arc dashboard](https://arc-dashboard.appbase.io/login).
+
+-   **Step 1 -** Sign in into [Arc dashboard](https://arc-dashboard.appbase.io/login) using your EC2 Public Domains / Domain with Username and Password configured above.
+
+![](https://i.imgur.com/qVSHx0F.png)
+
+🔥 Hurray! let's start accessing all Appbase services using [Arc Dashboard](https://arc-dashboard.appbase.io)
+
+---
+
 ### Using Docker
 
-[Arc](https://arc-site.netlify.com) is available as Docker Image as well, which can easily help you deploy and get started with your current docker based infrastructure. Based on your ElasticSearch version you can get a docker image and run it with required [Environment Variables](/docs/hosting/BringYourOwnCluster/#environment-variables).
+We have containerized Arc with its configuring dashboard into one docker compose file. This setup enables you to run Arc with single command, i.e. `docker-compose up -d` 😎.
 
-#### **Available Docker Images**
+Dockerize setup for Arc comes with 3 different services
 
-// TODO Add correct images links
+#### **Arc**
 
-| **ElasticSearch Version** | **Docker Image** |
-| ------------------------- | ---------------- |
-| ElasticSearch V7          | Docker Image     |
+Allows you to access all [Appbase.io](https://docs.appbase.io/docs/gettingstarted/WhyAppbase/) features like search preview, actionable analytics and granular security with any ElasticSearch cluster hosted anywhere.
 
-Before Deploying Arc Image, let's create an [Arc instance](https://arc-dashboard.appbase.io/install) which will give us access to [Arc Dashboard](https://arc-dashboard.appbase.io).
+> **Note:** Make sure your arc container have complete access to ElasticSearch. You can use ElasticSearch URL with Basic Auth in configuring dashboard or IP restricted ElasticSearch URL where IP of your ARC cluster hosted using docker setup is white listed.
 
-Here are the steps that you can follow to deploy Get Arc ID:
+#### **Configure**
 
--   **Step 1 -** Go to [Arc Dashboard](https://arc-dashboard.appbase.io/login) and Select [Install A New Arc Instance](https://arc-dashboard.appbase.io/install)
+This service comes with simple user interface which allows you to set credentials and other environment variables, that are required by the above service.
 
-![](https://i.imgur.com/YZubabh.png)
+#### **Watcher**
 
--   **Step 2 -** Enter your email address
+This services keeps eye on the environment file and restarts the arc service, whenever configurations are changed.
 
--   **Step 3 -** You will receive an OTP on entered email address. Enter OTP to verify the email address
+## Quick Start 🚀
```

Comment:
@lakhansamani It should be an h4 instead of an h2 to fit within the structure of Docker.